app/build.gradle

```diff
@@ -45,6 +46,19 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    // To run a different version next to standard version of the app uncomment flavorDimensions and productFlavors, comment 'client' in google-services.json, and select demo as build variant
```

Comment:
Please remove these commented out lines about the flavors and line changes related to flavors